src/cfnlint/data/AdditionalSpecs/Inclusive.json

```diff
@@ -65,30 +49,11 @@
       "VpcId": [
         "DefaultSubnetId"
       ]
+    },
+    "AWS::EC2::SecurityGroup": {
+      "SecurityGroupEgress": [
+        "VpcId"
+      ]
     }
-  },
-  "AWS::EC2::SecurityGroup": {
```

Comment:
These aren't in the right section.

Reply:
Not sure I follow exactly what you mean. Is there another section they should be in? Or just that they shouldn't be here?

Reply:
Some how it was out of resource types and property types.